### PR TITLE
[MM-32193][MM-32178] Move transform for team sidebar button to anchor

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
A CSS statement transforming the base team sidebar button container (`transform: translate(0, 0)`) was causing the desktop `Copy Link` context menu to not be rendered correctly.

Moving the transform to the `a` element inside the `team-container` stops it from affecting the context menu.

EDIT: Also noticed the extra transform statement was causing a horizontal scrollbar to appear. Seems to fix that as well.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-32193
https://mattermost.atlassian.net/browse/MM-32178

### CODE REVIEW

File: sass/layout/_team-sidebar.scss [hunk 1]
```diff
@@ -69,32 +62,23 @@
...
                 display: block;
                 text-decoration: none;
-            }
+                transition: transform 0.1s;
+                transform: translate(0, 0);
+                transform-origin: center center;
 
-            &.active {
-                .TeamIcon__content {
-                    @include opacity(1);
```
Comment (deanwhillier) [inline sass/layout/_team-sidebar.scss:97]: I think this still needs to be there. Unless you removed it separately from our conversation.

**Review by deanwhillier — CHANGES_REQUESTED** [review]
One accidental removal that I think needs to be brought back.

**Review by deanwhillier — APPROVED** [review]

**Review by jgilliam17 — APPROVED** [review]
Thanks @devinbinnie 
Tested, looks good to merge.
- Verified Team sidebar copy link is shown as expected.
- Verified Team sidebar no longer has a horizontal scroll bar on desktop app.

**Comment by nevyangelova** [comment]
@devinbinnie @jgilliam17 this was made so the badges can transition as well with the container when they are moved down on click. With this change I don't see the unread badges at all.

**Review by nevyangelova — CHANGES_REQUESTED** [review]
@devinbinnie @jgilliam17 this was made so the badges can transition as well with the container when they are moved down on click. With this change I don't see the unread badges at all.

**Review by hmhealey — APPROVED** [review]

**Comment by devinbinnie** [comment]
@nevyangelova I can see the unread badges just fine, there's a fix from @deanwhillier that fixes the bad CSS on the unread badges.

**Review by hmhealey — CHANGES_REQUESTED** [review]
In addition to Nevy's PR, that doesn't fix the horizontal scrollbar on long channels in the desktop app
Screen Shot 2021-01-21 at 9 24 18 AM

**Review by nevyangelova — CHANGES_REQUESTED** [review]
the unread badges disappear for me in this PR after clicking between the teams.

**Review by nevyangelova — APPROVED** [review]

**Review by hmhealey — APPROVED** [review]

**Pull request merged**